app.ts

```diff
@@ -163,10 +210,13 @@ export async function main() {
     dialerRegistry.registerDialers();
   }
 
+  const remoteDataSettings = parseRemoteDataSettings(manifestPath, radio);
+
   const simnet = await issueFirstClassCitizenship(
     manifestDir,
     manifestPath,
-    sutContractName
+    sutContractName,
+    remoteDataSettings
```

Comment:
If you move `remoteDataSettings` one position up, which also makes more sense as it's related to the manifest, you'll get a cleaner git diff here.